### DISCUSSION TITLE
Can we get a new release soon?

### DISCUSSION THREAD

**Discussion by black-sliver** [marcelotduarte/cx_Freeze#1576] [Q&A] [answered]
Current release version broke including tkinter on Linux. We currently downgrade cx_freeze by 1 version for the Linux builds in a project and the build automation depends on it coming from pypi.

I'm hopeful that https://github.com/marcelotduarte/cx_Freeze/pull/1526 fixes the problem (I have not tested it), so it'd be nice to get a new release in the next 2-4 weeks or so.

**Comment by marcelotduarte** [comment, marked as answer]
I believe it will be in that time frame.

If you can test with your project, to install the latest development build:

`pip install --pre --extra-index-url https://marcelotduarte.github.io/packages/ cx_Freeze`

**Reply by black-sliver** [reply]
Thanks!

Sadly there is still a problem with tk in 6.12.0.dev1. I'll open an issue for that.
https://github.com/marcelotduarte/cx_Freeze/issues/1577
